<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="46" passed="32" failed="14" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-09-12T20:44:24 CDT" name="Suite" finished-at="2022-09-12T20:45:29 CDT" duration-ms="64293">
    <groups>
      <group name="smoke">
        <method signature="Module2.test004()[pri:0, instance:testSuite1.Module2@7c137fd5]" name="test004" class="testSuite1.Module2"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2022-09-12T20:44:24 CDT" name="Test" finished-at="2022-09-12T20:45:29 CDT" duration-ms="64293">
      <class name="selenium.Assignment_5">
        <test-method signature="validateButton1()[pri:0, instance:selenium.Assignment_5@1033576a]" started-at="2022-09-12T20:44:45 CDT" name="validateButton1" finished-at="2022-09-12T20:44:52 CDT" duration-ms="6997" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateButton1 -->
        <test-method signature="validateButton2()[pri:0, instance:selenium.Assignment_5@1033576a]" started-at="2022-09-12T20:44:52 CDT" name="validateButton2" finished-at="2022-09-12T20:45:00 CDT" duration-ms="7505" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateButton2 -->
        <test-method signature="validateTexbox1()[pri:0, instance:selenium.Assignment_5@1033576a]" started-at="2022-09-12T20:45:00 CDT" name="validateTexbox1" finished-at="2022-09-12T20:45:11 CDT" duration-ms="10758" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTexbox1 -->
        <test-method signature="validateTextAreaField()[pri:0, instance:selenium.Assignment_5@1033576a]" started-at="2022-09-12T20:45:11 CDT" name="validateTextAreaField" finished-at="2022-09-12T20:45:18 CDT" duration-ms="7468" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTextAreaField -->
        <test-method signature="validateWebPageload()[pri:0, instance:selenium.Assignment_5@1033576a]" started-at="2022-09-12T20:45:18 CDT" name="validateWebPageload" finished-at="2022-09-12T20:45:29 CDT" duration-ms="10597" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateWebPageload -->
      </class> <!-- selenium.Assignment_5 -->
      <class name="selenium.SeleniumBasicTest">
        <test-method signature="validateSeleniumSetup()[pri:0, instance:selenium.SeleniumBasicTest@247310d0]" started-at="2022-09-12T20:44:25 CDT" name="validateSeleniumSetup" finished-at="2022-09-12T20:44:45 CDT" duration-ms="20287" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateSeleniumSetup -->
      </class> <!-- selenium.SeleniumBasicTest -->
      <class name="testSuite1.Module1">
        <test-method is-config="true" signature="BeforeSuiteMethod()[pri:0, instance:testSuite1.Module1@24569dba]" started-at="2022-09-12T20:44:24 CDT" name="BeforeSuiteMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="24" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeSuiteMethod -->
        <test-method signature="test001()[pri:0, instance:testSuite1.Module1@24569dba]" started-at="2022-09-12T20:44:24 CDT" name="test001" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test001 -->
        <test-method signature="test002()[pri:0, instance:testSuite1.Module1@24569dba]" started-at="2022-09-12T20:44:24 CDT" name="test002" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test002 -->
        <test-method signature="test003()[pri:0, instance:testSuite1.Module1@24569dba]" started-at="2022-09-12T20:44:24 CDT" name="test003" finished-at="2022-09-12T20:44:24 CDT" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test003 -->
      </class> <!-- testSuite1.Module1 -->
      <class name="testSuite1.Module2">
        <test-method is-config="true" signature="BeforeSuiteMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="BeforeSuiteMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeSuiteMethod -->
        <test-method is-config="true" signature="BeforeTestmethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="BeforeTestmethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeTestmethod -->
        <test-method is-config="true" signature="BeforeClassMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="BeforeClassMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeClassMethod -->
        <test-method is-config="true" signature="BeforeMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="BeforeMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeMethodMethod -->
        <test-method signature="test001()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="test001" finished-at="2022-09-12T20:44:24 CDT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test001 -->
        <test-method is-config="true" signature="AfterMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="AfterMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterMethodMethod -->
        <test-method is-config="true" signature="BeforeMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="BeforeMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeMethodMethod -->
        <test-method signature="test002()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="test002" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test002 -->
        <test-method is-config="true" signature="AfterMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="AfterMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterMethodMethod -->
        <test-method signature="test003()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="test003" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test003 -->
        <test-method is-config="true" signature="BeforeMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="BeforeMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeMethodMethod -->
        <test-method is-config="true" signature="AfterMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="AfterMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterMethodMethod -->
        <test-method is-config="true" signature="beforeGroup()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="beforeGroup" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeGroup -->
        <test-method signature="test004()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="test004" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test004 -->
        <test-method is-config="true" signature="BeforeMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="BeforeMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeMethodMethod -->
        <test-method is-config="true" signature="AfterMethodMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="AfterMethodMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterMethodMethod -->
        <test-method is-config="true" signature="AfterGroup()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="AfterGroup" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterGroup -->
        <test-method is-config="true" signature="AfterClassMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:44:24 CDT" name="AfterClassMethod" finished-at="2022-09-12T20:44:24 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterClassMethod -->
        <test-method is-config="true" signature="AfterTestMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:45:29 CDT" name="AfterTestMethod" finished-at="2022-09-12T20:45:29 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTestMethod -->
        <test-method is-config="true" signature="AfterSuiteMethod()[pri:0, instance:testSuite1.Module2@7c137fd5]" started-at="2022-09-12T20:45:29 CDT" name="AfterSuiteMethod" finished-at="2022-09-12T20:45:29 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterSuiteMethod -->
      </class> <!-- testSuite1.Module2 -->
      <class name="testSuite2.EmployeeSalaryTest">
        <test-method signature="ValidateEmpSal()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:24 CDT" name="ValidateEmpSal" finished-at="2022-09-12T20:44:24 CDT" duration-ms="12" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [5000.0] but found [5000.001]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [5000.0] but found [5000.001]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at org.testng.Assert.assertEquals(Assert.java:879)
at testSuite2.EmployeeSalaryTest.ValidateEmpSal(EmployeeSalaryTest.java:90)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateEmpSal -->
        <test-method signature="validateEmpDiv()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:24 CDT" name="validateEmpDiv" finished-at="2022-09-12T20:44:24 CDT" duration-ms="0" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Devloper] but found [Devloper.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Devloper] but found [Devloper.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testSuite2.EmployeeSalaryTest.validateEmpDiv(EmployeeSalaryTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpDiv -->
        <test-method signature="validateEmpDiv1(java.lang.String)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpDiv1" data-provider="dataEmpDiv" finished-at="2022-09-12T20:44:25 CDT" duration-ms="519" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Tester]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Tester] but found [Tester.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Tester] but found [Tester.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpDiv1 -->
        <test-method signature="validateEmpDiv1(java.lang.String)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpDiv1" data-provider="dataEmpDiv" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Dev]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dev] but found [Dev.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dev] but found [Dev.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpDiv1 -->
        <test-method signature="validateEmpDiv1(java.lang.String)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpDiv1" data-provider="dataEmpDiv" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Pwoner]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Pwoner] but found [Pwoner.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Pwoner] but found [Pwoner.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpDiv1 -->
        <test-method signature="validateEmpDiv1(java.lang.String)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpDiv1" data-provider="dataEmpDiv" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Dir]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Dir] but found [Dir.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Dir] but found [Dir.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpDiv1 -->
        <test-method signature="validateEmpDiv1(java.lang.String)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpDiv1" data-provider="dataEmpDiv" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[VP]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [VP] but found [VP.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [VP] but found [VP.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testSuite2.EmployeeSalaryTest.validateEmpDiv1(EmployeeSalaryTest.java:176)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpDiv1 -->
        <test-method signature="validateEmpId()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpId" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpId -->
        <test-method signature="validateEmpId1()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpId1" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpId1 -->
        <test-method signature="validateEmpName()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shardul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shardul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shubhangi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shubhangi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shubhangi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shardul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shardul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shardul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shubhangi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shardul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shubhangi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shubhangi]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[122]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpName1(java.lang.String,int)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpName1" data-provider="dataForEmpName" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Shardul]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpName1 -->
        <test-method signature="validateEmpNm()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpNm" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpNm -->
        <test-method signature="validateEmpSalDP(float)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpSalDP" data-provider="dataEmpSal" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[500.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [500.0] but found [500.001]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [500.0] but found [500.001]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at org.testng.Assert.assertEquals(Assert.java:879)
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpSalDP -->
        <test-method signature="validateEmpSalDP(float)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpSalDP" data-provider="dataEmpSal" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[501.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [501.0] but found [501.001]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [501.0] but found [501.001]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at org.testng.Assert.assertEquals(Assert.java:879)
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpSalDP -->
        <test-method signature="validateEmpSalDP(float)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpSalDP" data-provider="dataEmpSal" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[502.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [502.0] but found [502.001]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [502.0] but found [502.001]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at org.testng.Assert.assertEquals(Assert.java:879)
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpSalDP -->
        <test-method signature="validateEmpSalDP(float)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpSalDP" data-provider="dataEmpSal" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[503.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [503.0] but found [503.001]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [503.0] but found [503.001]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at org.testng.Assert.assertEquals(Assert.java:879)
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpSalDP -->
        <test-method signature="validateEmpSalDP(float)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpSalDP" data-provider="dataEmpSal" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[504.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [504.0] but found [504.001]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [504.0] but found [504.001]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at org.testng.Assert.assertEquals(Assert.java:879)
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpSalDP -->
        <test-method signature="validateEmpSalDP(float)[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateEmpSalDP" data-provider="dataEmpSal" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[505.0]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [505.0] but found [505.001]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [505.0] but found [505.001]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEquals(Assert.java:844)
at org.testng.Assert.assertEquals(Assert.java:879)
at testSuite2.EmployeeSalaryTest.validateEmpSalDP(EmployeeSalaryTest.java:155)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateEmpSalDP -->
        <test-method signature="validateMaxInt()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateMaxInt" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateMaxInt -->
        <test-method signature="validateMinValue()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateMinValue" finished-at="2022-09-12T20:44:25 CDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateMinValue -->
        <test-method signature="validateempDiv()[pri:0, instance:testSuite2.EmployeeSalaryTest@7c37508a]" started-at="2022-09-12T20:44:25 CDT" name="validateempDiv" finished-at="2022-09-12T20:44:25 CDT" duration-ms="0" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ Product Owner] but found [ Product Owner.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ Product Owner] but found [ Product Owner.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testSuite2.EmployeeSalaryTest.validateempDiv(EmployeeSalaryTest.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateempDiv -->
      </class> <!-- testSuite2.EmployeeSalaryTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
